src/std/crypto/libcrypto.ss

```diff
@@ -507,5 +479,138 @@ static int ffi_DH_compute_key (___SCMOBJ secret, BIGNUM *pubkey, DH *dh)
  return DH_compute_key (U8_DATA (secret), pubkey, dh);
 }
 
+#if OPENSSL_VERSION_NUMBER < 0x10002000L || defined (LIBRESSL_VERSION_NUMBER)
+#include "crypto/libcrypto-rfc5114.c"
```

Comment:
why is this in `crypto/` and not straight up include? It's in the same damn directory!
This will still break the static exe compilation I think.

Reply:
Fixed.